python/example/single_cell_cable.py

```diff
@@ -72,6 +70,13 @@ def cell_kind(self, gid):
         assert gid == 0
         return arbor.cell_kind.cable
 
+    def probes(self, gid):
+        assert gid == 0
+        return self.the_probes
+
+    def global_properties(self, kind):
```

Comment:
Can we?

I think the asserts in this file might as well be removed, since I can't think of how `gid>0` can ever be interogated when `num_cells == 1`.

Reply:
Jupp, that's fine, too. It also confuses people.